apps/frontend/src/libs/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,72 @@
+import { useCallback, useFormController } from "~/libs/hooks/hooks.js";
+import {
+	type CheckboxInputOption,
+	type Control,
+	type FieldPath,
+	type FieldValues,
+} from "~/libs/types/types.js";
+
+import styles from "./styles.module.css";
+
+type Properties<T extends FieldValues> = {
+	control: Control<T, null>;
+	label: string;
+	name: FieldPath<T>;
+	options?: CheckboxInputOption[];
```

Comment:
```suggestion
	options?: InputOption[];
```
Since we have the same for Checkbox and Radio, we decided to combine

Reply:
But I have a problem with combine this types. In InputOption value:string, in CheckboxInputOption value:number. Should I change option data in checkbox to string? Because now it's 
```
const TASK_DAYS_OPTIONS = [
	{ label: "Monday", value: 1 },
	{ label: "Tuesday", value: 2 },
```

Reply:
Change InputOption value type to accept `string | number` and use type assertion where needed